backend/src/main/java/com/ody/notification/domain/FcmTopic.java

```diff
@@ -0,0 +1,25 @@
+package com.ody.notification.domain;
+
+import com.ody.meeting.domain.Meeting;
+import lombok.Getter;
+
+@Getter
+public class FcmTopic {
+
+    private String value;
+
+    public FcmTopic(String rawValue) {
+        this.value = rawValue.replace(":", "-");
+    }
+
+    public FcmTopic(Meeting meeting) {
+        this(build(meeting));
+    }
```

Comment:
[제안]

생성자 체이닝 시에는 주 생성자가 부 생성자 아래에 위치하는게 컨벤션으로 알고있어요!

```suggestion
    public FcmTopic(Meeting meeting) {
        this(build(meeting));
    }
    
    public FcmTopic(String rawValue) {
        this.value = rawValue.replace(":", "-");
    }
```

Reply:
[반영] [629a911](https://github.com/woowacourse-teams/2024-ody/pull/367/commits/629a911a66c1e926a531bac1da8266586a19b056)